app/main.py

```diff
@@ -1,6 +1,37 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv("API_KEY")
+
+if not API_KEY:
+    raise ValueError("API_KEY is not found.")
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    params = {
+        "key": API_KEY,
+        "q": CITY
+    }
+    res = requests.get(BASE_URL, params=params)
+    if res.status_code == 200:
+        data = res.json()
+        location = data["location"]
+
+        print(f"Taking weather information about {CITY} from Weather API")
+        print(
+            f"Location: {location['name']}, {location['country']}\n"
+            f"Time: {location['localtime']}\n"
+            f"Temperature: {data['current']['temp_c']} Celsius degrees"
```

Comment:
There is a syntax error in the `print` statement due to incorrect usage of double quotes within an f-string. You should use single quotes around the dictionary keys or escape the double quotes to avoid this error. For example, use `f"Location: {location['name']}, {location['country']}\n"`.